docs/static/logging.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[logging]]
+== Logstash's Internal Logging
```

Comment:
I'd be inclined to just to call this topic "Logging" (saying "internal logging" might suggest to users that it's not something they can configure.)

Let me know where you want to add this section. I think it makes sense under "Setting Up and Running Logstash".